misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+<svg version="1.1" viewBox="0 0 1000 1350" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .external{
+        stroke-dasharray: 3pt;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .left-rotation{
+        transform: rotate(-90deg);
+      }
+      .right-rotation{
+        transform: rotate(90deg);
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+      .relation.to-many {
+        marker-end: url(#relation-many)
+      }
+      .relation.from-optional {
+        marker-start: url(#relation-optional)
+      }
+      .relation.to-optional {
+        marker-end: url(#relation-optional)
+      }
+      .relation.to-link_attribute {
+        marker-end: url(#link_attribute)
+      }
+      .dependency {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+        marker-end: url(#dependency);
+      }
+  </style>
+
+  <defs>
+    <marker id="relation-many" orient="auto-start-reverse" markerWidth="6" markerHeight="6" refX="6" refY="3">
+      <circle cx="3" cy="3" r="3" />
+    </marker>
+    <marker id="relation-optional" orient="auto-start-reverse" markerWidth="6" markerHeight="6" refX="6" refY="3">
+      <circle cx="3" cy="3" r="2.5" fill="white" stroke="black" stroke-width="1" />
+    </marker>
+    <marker id="link_attribute" orient="-90" markerWidth="10" markerHeight="6" refX="5" refY="0">
```

Comment:
@martinmacko47 ten polkruh nie je symetricky ako kruh, takze ho nemozem lubovolne otacat. rovnako plati pre trojuholnik.

Reply:
Ale samozrejme, ze ho mozes otacat aj nesymetricke markery, ved na to sluzi `orient="auto-start-reverse"`:

![marker](https://user-images.githubusercontent.com/5547275/64103600-df6ded80-cd72-11e9-9009-61d458cc2044.png)

Len path treba prepisat tak, aby to bolo spravne natocene pri spravnom uhle.

Reply:
@martinmacko47. Takze ak tomu správne rozumiem, koniec arc bude natočený v tomto prípade nahor? Do `d` som pridal `-200,-1`
```    
<path class="relation from-one to-link-attribute" d="M -100,0 -200,0 -200,-1" />
```
![Screenshot from 2019-09-03 11-03-32](https://user-images.githubusercontent.com/29918752/64159473-9da87a00-ce3a-11e9-8722-2f413489a816.png)

Reply:
Neviem ake presne su uhly natocenia pre start a end markery. Ale tipujem, ze 0 a 180 alebo naopak. Samotny marker treba nakreslit tak, ze pri natoceni "start" je obluk pri ciare. Takze ked sa pri "end" otoci o 180 tak obluk tiez bude pri ciare.

Ved na to je presne `auto-start-reverse`, aby fungovali sipky. Takze to musi fungovat, len musis spravne otocenu tu sipku nakreslit. Neviem ako ti to lepsie vysvetlit.

Reply:
ano dakujem, uz tomu rozumiem, 
opravene :ok: